Makefile

```diff
@@ -22,6 +22,18 @@ endif
 # Build Flags
 GO_LD_FLAGS= "-s -w -X github.com/megaease/easegress/pkg/version.RELEASE=${RELEASE} -X github.com/megaease/easegress/pkg/version.COMMIT=${GIT_COMMIT} -X github.com/megaease/easegress/pkg/version.REPO=${GIT_REPO_INFO}"
 
+# Cgo is disabled by default
+ENABLE_CGO= CGO_ENABLED=0
+
+# Check Go build tags, the tags are from command line of make
+ifdef GOTAGS
```

Comment:
Actually, I don't think it's a big deal to include more than 20MB in EG with wasm. I think it brings more concern in compilation, which gives more confusing stuff to developers, operators.